projects/plugins/jetpack/modules/masterbar/admin-menu/class-wpcom-admin-menu.php

```diff
@@ -395,26 +395,6 @@ public function should_link_to_wp_admin() {
 		return $result;
 	}
 
-	/**
-	 * Adds Plugins menu.
-	 */
-	public function add_plugins_menu() {
-		// TODO: Remove wpcom_menu (/wp-content/admin-plugins/wpcom-misc.php).
-		$count = '';
-		if ( ! is_multisite() && current_user_can( 'update_plugins' ) ) {
-			$update_data = wp_get_update_data();
-			$count       = sprintf(
-				'<span class="update-plugins count-%s"><span class="plugin-count">%s</span></span>',
-				$update_data['counts']['plugins'],
-				number_format_i18n( $update_data['counts']['plugins'] )
-			);
-		}
```

Comment:
This is some logic that we copy/pasted from a file that was shared between Simple and Atomic sites. For Simple sites, we don't need this logic (`is_multisite` always returns true), which is why I didn't incorporated into the jetpack-mu-wpcom package.